internal/process/cluster.go

```diff
@@ -66,6 +67,33 @@ func createClusterSamples(config *load.Config) {
 			checkError(config, ms.SetMetric("effectiveHosts", summary.NumEffectiveHosts, metric.GAUGE))
 			checkError(config, ms.SetMetric("hosts", summary.NumHosts, metric.GAUGE))
 
+			//DRS metrics
+			if cluster.Configuration.DrsConfig.Enabled != nil {
+				checkError(config, ms.SetMetric("drsConfig.Enabled", strconv.FormatBool(*cluster.Configuration.DrsConfig.Enabled), metric.ATTRIBUTE))
```

Comment:
please mod metrics name to follow the camelCase.camelCase. Thanks!

Reply:
done